bundles/core/src/test/resources/form/title/exporter-title-noprops.json

```diff
@@ -0,0 +1,6 @@
+{
+  "id": "title-826fc94cbc",
+  "type": "h2",
```

Comment:
I dont think type: "h2" is supported in AF2 schema. cc @vdua

Reply:
It is the heading type, h1, h2, h3, h4, h5, h6

Reply:
This does not have fieldType, this means it is not a form component. Hence the schema could be anything